common.gypi

```diff
@@ -407,12 +410,17 @@
         'libraries': [ '-lelf' ],
       }],
       ['OS=="freebsd"', {
-        # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial
-        # https://lists.freebsd.org/pipermail/freebsd-toolchain/2016-March/002094.html
-        'cflags': [ '-D_LIBCPP_TRIVIAL_PAIR_COPY_CTOR=1' ],
         'ldflags': [
           '-Wl,--export-dynamic',
         ],
+        'conditions': [
+          ['clang_major_version<4', {
+            # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial
```

Comment:
nits:
1. put a `.` at the end of the two sentences
2. put them one after the other
3. put the URL last and start the line with `Ref: https://...`